vod/mp4/mp4_parser.c

```diff
@@ -278,6 +285,11 @@ mp4_parser_parse_hdlr_atom(atom_info_t* atom_info, metadata_parse_context_t* con
 		return VOD_OK;
 	}
 
+	// already set
+	if (context->media_info.label.data != NULL) {
+		return VOD_OK;
+	}
+
```

Comment:
1. the convention for opening braces here is to have them a new line
2. probably better to add this condition with || on the previous if

Reply:
1. I noticed but forgot about it when pushing... I am used to just throw clang-format at stuff after I'm done 😋
2. done